Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -2182,6 +2184,7 @@ ModelExperimental.fromB3dm = function (options) {
     arrayBuffer: options.arrayBuffer,
     byteOffset: options.byteOffset,
     releaseGltfJson: options.releaseGltfJson,
+    asynchronous: options.asynchronous,
```

Comment:
Can this go in `makeModelOptions()`? is anything else common enough to move to that function? I think I've been forgetting to check.

Reply:
@ptrgags I don't think it can because the `asynchronous` field needs to be passed to the loaders (`B3dmLoader, I3dmLoader`). It doesn't actually get passed to the `ModelExperimental` constructor.

Reply:
:facepalm: you're right, my bad.